### PR TITLE
Improved the hazard map storage and export

### PR DESCRIPTION
**Pull request by micheles**
Closes https://github.com/gem/oq-risklib/issues/599. The tests are green on Jenkins too: https://ci.openquake.org/job/zdevel_oq-risklib/1285/

### CODE REVIEW

File: openquake/qa_tests_data/classical/case_15/expected/hazard_uhs-mean-0.01.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@ xmlns:gml="http://www.opengis.net/gml"
                 </gml:pos>
             </gml:Point>
             <IMLs>
-                0.2 0.8
+                0.8 0.8
```
Comment (raoanirudh) [inline openquake/qa_tests_data/classical/case_15/expected/hazard_uhs-mean-0.01.xml]: Expected values seem to have changed here.

File: openquake/qa_tests_data/classical/case_15/expected/hazard_uhs-mean.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
 lon,lat,poe~0.01:2,poe~0.1:2,poe~0.2:2
--1.00000001E-01,0.00000000E+00,2.00000003E-01 8.00000012E-01,2.00000003E-01 8.00000012E-01,2.00000003E-01 6.34829700E-01
-0.00000000E+00,0.00000000E+00,2.00000003E-01 8.00000012E-01,2.00000003E-01 8.00000012E-01,2.00000003E-01 5.94248533E-01
-1.00000001E-01,0.00000000E+00,2.00000003E-01 8.00000012E-01,2.00000003E-01 8.00000012E-01,2.00000003E-01 7.47483134E-01
+-1.00000001E-01,0.00000000E+00,8.00000012E-01 8.00000012E-01,8.00000012E-01 8.00000012E-01,6.34829700E-01 6.34829700E-01
```
Comment (raoanirudh) [inline openquake/qa_tests_data/classical/case_15/expected/hazard_uhs-mean.csv]: Similar comment as above; expected values for poe~0.01 seem to have changed here.

File: openquake/qa_tests_data/classical/case_15/expected/hazard_uhs-mean-0.01.xml (hunk 1 above)
Comment (micheles) [inline openquake/qa_tests_data/classical/case_15/expected/hazard_uhs-mean-0.01.xml]: Indeed, there was a bug in `make_uhs` which is fixed now.

**Comment by raoanirudh** [comment]
LGTM

**Pull request merged**